<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>160</width>
     <height>165</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameInput">
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameInput">
      <property name="placeholderText">
       <string>Nome de usu√°rio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordInput">
      <property name="placeholderText">
       <string>Senha mestre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signupButton">
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
